src/i2c/src/i2c.rs

```diff
@@ -23,10 +111,237 @@ pub struct I2cBackend {
     adapter_map: [u32; MAX_I2C_VDEV],
 }
 
+fn vi2c_set_client_addr(fd: i32, addr: usize) -> bool {
```

Comment:
It is a bit atypical to return a boolean. Can we return an Error instead? It looks like this is a wrapper over the `I2C_SLAVE` ioctl, so we could forward the `errno` instead.

This applies to all functions that now return a bool instead of `Result`. For some example, you can check how this is handled in the `kvm-ioctls` crate.

Reply:
Sure